CollAction/Helpers/InputSanitizer.cs

```diff
@@ -0,0 +1,20 @@
+using Ganss.XSS;
+
+namespace CollAction.Helpers
+{
+    public static class InputSanitizer
+    {
+        public static string Sanitize(string input)
+        {
+            var saniziter = new HtmlSanitizer(
+                allowedTags: new[] { "p", "br", "strong", "em", "i", "u", "a", "ol", "ul", "li" },
+                allowedAttributes: new[] { "href", "target" },
+                allowedCssClasses: new string[] {},
+                allowedCssProperties: new string[] {}                    
+            );
+
+            return saniziter.Sanitize(input);
+
```

Comment:
Unnecessary empty line. Also, might be cleaner with a validation attribute: https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-2.1

Reply:
Also, the sanitizing might be a bit simplistic. For example, I think this will still get through:

<a href="javascript:alert(8007)">Click me</a>

You need to verify the contents of those attributes as well (for example, is the href a valid url with protocol http or https).